data/sql/updates/pending_db_world/rev_1621501368347745883.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1621501368347745883');
+
+UPDATE `creature`
+SET `modelid` = 1744, `spawntimesecs` = 275, `wander_distance` = 5, `curhealth` = 178, `movementtype` = 1
+WHERE `guid` BETWEEN 40489 AND 40496;
```

Comment:
AND id =

Reply:
is that actually needed? `id` isn't primary key on `creature`

Reply:
It is preferred just in case someone did some custom thing.

This way you know you will only make changes to the correct creature as `id` can't be a primary or it would be wild :D

Reply:
ah alright. that makes sense